statsmodels/stats/rates.py

```diff
@@ -20,41 +20,41 @@ def test_poisson_2indep(count1, exposure1, count2, exposure2, ratio_null=1,
 
     If the two Poisson rates are g1 and g2, then the Null hypothesis is
 
-    H0: g1 / g2 = ratio_null
+    - H0: g1 / g2 = ratio_null
 
     against one of the following alternatives
 
-    H1_2-sided: g1 / g2 != ratio_null
-    H1_larger: g1 / g2 > ratio_null
-    H1_smaller: g1 / g2 < ratio_null
+    - H1_2-sided: g1 / g2 != ratio_null
+    - H1_larger: g1 / g2 > ratio_null
+    - H1_smaller: g1 / g2 < ratio_null
 
     Parameters
     ----------
-    count1 : int
-        Number of events in first sample
-    exposure1 : float
-        Total exposure (time * subjects) in first sample
-    count2 : int
-        Number of events in first sample
-    exposure2 : float
-        Total exposure (time * subjects) in first sample
-    ratio : float
+    count1: int
```

Comment:
Missing space after variable names. These are required for docs to render correctly.